churchslavonic.tex

```diff
@@ -3,6 +3,9 @@
 
 \def\pkg#1{\textsf{#1}}
 \def\cs#1{\texttt{\textbackslash #1}}
+\providecommand{\XeTeX}{X\kern-.125em\lower.5ex\hbox{Ǝ}\kern-.1667em\TeX}
+\providecommand{\XeLaTeX}{X\kern-.125em\lower.5ex\hbox{Ǝ}\kern-.125em\LaTeX}
+\providecommand{\LuaTeX}{Lua\kern-.125em\TeX}
```

Comment:
On my computer, make errors. Does churchlsavonic.tex actually load xltxtra? I don't see it in the preamble.